pipelines/build-release.yml

```diff
@@ -13,7 +13,7 @@ resource_types:
   - name: pull-request
     type: docker-image
     source:
-      repository: governmentpaas/pullrequest-resource
+      repository: jtarchie/pr
```

Comment:
is this a trustable, permanent source of the resource? or shall we try to bring it to governmentpaas?